website/publish/domain_name.rst

```diff
@@ -81,6 +81,15 @@ We can now apply the redirection from your domain name's manager account:
    - `Namecheap <https://www.namecheap.com/support/knowledgebase/article.aspx/9646/10/how-can-i-set-up-a-cname-record-for-my-domain>`__
    - `OVH <https://www.ovh.co.uk/g1519.exchange_20132016_how_to_add_a_cname_record>`__
 
+Impact on email deliverability
+==============================
+
+If your domain is configurable and you are using email addresses associate with it, you might need to configure it to avoid your email to be received in spam.
```

Comment:
```suggestion
..warning:: If you are using email addresses associated with your custom domain name, you must take care of a few additional steps. Otherwise, your emails have good chances to end up in the *Spam* folder.